client/src/controllers/search.controller.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+app.controller('SearchCtrl', ['$scope', 'SunfishSrvc', function($scope, SunfishSrvc){
+    $scope.search = function() {
+        SunfishSrvc.searchSiafiles($scope.query)
+          .success(function(siafiles){
+              $scope.siafiles = siafiles;
```

Comment:
idk if this is callback hell...  declaring a function that calls search and handles the result?